spec/unit/version_spec.lua

```diff
@@ -6,16 +6,12 @@ describe("Version module", function()
     end)
     it("should get current revision", function()
         local rev = Version:getCurrentRevision()
-        local year, month, revision = rev:match("v(%d%d%d%d)%.(%d%d)-?(%d*)")
-        local commit = rev:match("-%d*-g(%x*)[%d_%-]*")
+        local year, month, point, revision = rev:match("v(%d%d%d%d)%.(%d%d).(%d?)-?(%d*)") -- luacheck: ignore 211
```

Comment:
@NiLuJe On the flipside, it's probably just the `.1` that was tripping it up.

Reply:
I mean, in a PR here on GH it'll presumably always be v2018.11.1-1-gasdafsdf at the very lowest, right?